docs/UserGuide.md

```diff
@@ -142,6 +142,20 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
+### Viewing a student : `view`
+
+Displays the specified student’s name, phone number, progress and payment status, along with their parent’s name and phone number.
+
+Format: `view INDEX`
```

Comment:
Maybe can change `INDEX` to `STUDENT_INDEX` so that it is clearer? 

Otherwise LGTM

Reply:
Yes, have changed it. Thank you.